generators/ionic/resources/base/package.json

```diff
@@ -12,48 +12,11 @@
     "ng": "ng",
     "prettier": "prettier --write \"{,e2e/**/,src/**/}*.{js,json,html,md,ts,css,scss,yml}\" --loglevel silent",
     "start": "ionic serve",
-    "test": "jest",
-    "test:ci": "jest --runInBand",
-    "test:coverage": "jest --coverage",
-    "test:watch": "jest --watch",
+    "test": "ng test --coverage",
+    "test:watch": "ng test --watch",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run"
   },
-  "jest": {
-    "collectCoverageFrom": [
-      "src/app/**/*.{js,ts}",
-      "!src/app/**/index.ts",
-      "!src/app/**/*.module.ts",
-      "!<rootDir>/node_modules/"
-    ],
-    "coverageDirectory": "./public/coverage",
```

Comment:
Maybe we should create a `jest.config.js` with this one setting. That would allow us to get rid of the warning that prints at the beginning and make it so we don't need to add `coverage/` to `.gitignore`.